src/main/java/com/sarahhand/fractals/juliaset/JuliaViewer.java

```diff
@@ -217,7 +217,9 @@ public JuliaViewer(FractalConfig config){
 	
 	private Color getPointCol(Double p){
 		
-		ComplexNumber c = new ComplexNumber(0.37259789000000026, 0.0909054899999998);
+		ComplexNumber c = new ComplexNumber(config.getConstants().get("c real"), config.getConstants().get("c imaginary"));
```

Comment:
Make "c real" and "c imaginary" constants and use them instead of the hardcoded value.